chrome/locale/en-US/zotero/preferences.dtd

```diff
@@ -107,6 +107,7 @@
 <!ENTITY zotero.preferences.quickCopy.siteEditor.domainPath "Domain/Path">
 <!ENTITY zotero.preferences.quickCopy.siteEditor.domainPath.example "(e.g. wikipedia.org)">
 <!ENTITY zotero.preferences.quickCopy.siteEditor.outputFormat "Output Format">
+<!ENTITY zotero.preferences.quickCopy.siteEditor.locale		"Locale">
```

Comment:
Done: https://github.com/rmzelle/zotero/commit/2e10f16aa1dd8f1041a2496f62741e714681a224 . I also changed the string used in the quickCopySiteEditor, which had a colon, which was inconsistent.